modules/ocf/manifests/ssl/setup.pp

```diff
@@ -0,0 +1,45 @@
+# Provide the common certificates needed at the OCF.
+# To build and deploy private keys, use the `ocf::ssl::bundle` type.
+# (Or include `ocf::ssl::default` if you just want the default one that includes
+# the FQDN and all aliases of a server.)
+class ocf::ssl::setup {
+  package { 'ssl-cert':; }
+
+  file {
+    default:
+      # the ssl-cert package creates the ssl-cert group
+      require => Package['ssl-cert'];
```

Comment:
No, this was migrated from `ocf_ssl`, it's necessary because I don't believe `/etc/ssl/certs` exists without the `ssl-cert` package already being installed, but you're right that the file at the bottom of this block can have the `require` removed, I copy/pasted it from another manifest so that's why the `require` is duplicated.

Reply:
The comment on this needed updating though, since it didn't explain the directories being created, so I see now why you would have thought it was just needed for the `ssl-cert` group.